src/commands/login.js

```diff
@@ -41,14 +41,14 @@ Cypress.Commands.add('login', (credentials = {}) => {
 
             /* https://github.com/auth0/nextjs-auth0/blob/master/src/session/cookie-store/index.ts#L73 */
 
-            cy.task('encrypt', payload).then((encryptedSession) => {
-              cy.setCookie(sessionCookieName, encryptedSession);
-            })
+            cy.task('encrypt', payload).then(encryptedSession => {
+              cy._handleAuth0Cookie(encryptedSession);
```

Comment:
I think the naming convention makes sense, since this command is intended for private use only. However, if I was reading this code for the first time I might not realize than `_handleAuth0Cookie()` also sets the cookie. I'm curious if you think `_handleAuth0Cookie()` should be named, perhaps, `_setSplitCookie()` or something like that?

Reply:
Good idea, thanks for the feedback. I've updated it _and_ added a new public command that helps to mimic expiring cookies without visiting the logout URLs (`cy.clearAuth0Cookies`).

This also resolves https://github.com/sir-dunxalot/cypress-nextjs-auth0/issues/29. 😅 (added to the list above).

see https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44/commits/cba4098a04c522b23195d205617a789680d1b8ad